docs/effects/particles.md

```diff
@@ -0,0 +1,85 @@
+Particles
+=========
+
+Terminology
+-----------
+| Term | Description |
+|----------------|----------------|
+| Particle | What is responsible for the rendering logic. |
+| ParticleType<?> | What is registered using a unique name. |
+| IParticleData | What contains server data that needs to be transferred to the client. |
+| IParticleFactory | The factory takes in an IParticleData and returns a Particle. <br> It will be registered using a unique ParticleType. |
+
+Different Particles
+-------------------
+There are 4 abstract particle classes: Particle, TexturedParticle,  SpriteTexturedParticle and SimpleAnimatedParticle. Each inherit from the previous and your own Particle has to extend one of these.
+In it, the rendering of the Particle is handled. I suggest looking at vanilla examples on how to handle rendering, this guide will not get into that, only the logic part of Particles.
+
+### Sidedness
+
+The Particle class is [**client only**][sides], it does not exist on the server.
+This means that creating a fixed parameter particle (no context information needed) is somewhat simpler, more on that later. 
+However, if the particle has a parameter with continuous values, server information is required. For example, breaking blocks create a particle effect based on the broken block's texture.
+This information obviously depends on the BlockState of the block, which is taken from the server. <br>
+When adding a particle on the client, use ClientWorld#addParticle, but on the server it is ServerWorld#spawnParticle. ServerWorld#addParticle will simply do nothing (careful, one overload of the method in World is ClientOnly). 
+
+### Does your particle need server data?
+Say you want to emit a particle on spell use. If you have 8 type of spells with each one specific color no need for server information. Each spell will have a distinct registered ParticleType. 
+However, if these spells can have modifiers that can affect the color in multiple ways (add/subtract red/green/blue data) then having a registry object for each combination will be fastidious. 
```

Comment:
I'm not sure "fastidious" makes sense here.

Reply:
yeah my bad, its similar to a word in French in pronunciation but not in meaning, changed to tedious.